src/perfcollect/perfcollect

```diff
@@ -634,6 +639,9 @@ declare logFile
 logFilePrefix='/tmp/perfcollect'
 logEnabled=1
 
+# Collect info to pass between processes
+collectInfoFile='collectinfo'
```

Comment:
Can you rename this to something like "perfcollect.sessioninfo" so that it's clear if someone runs across this what it is?